project/newEvaluation.html

```diff
@@ -1,472 +1,489 @@
-                <option value="0">0</option>
-                <option value="1">1</option>
-                <option value="2">2</option>
-                <option value="3">3</option>
-              </select>
-            </li>
-            <li>
-              <label for="frame">Frameworks</label>
-              <select name="frame" id="brow">
-                <option value="0">0</option>
-                <option value="1">1</option>
-                <option value="2">2</option>
-                <option value="3">3</option>
-              </select>
-            </li>
-        </ul>
-      </fieldset>
+          <fieldset>
+            <legend>Javascript</legend>
+            <ul>
+              <li>
+                  <label for="data">Data types and variables</label>
+                    <select name="data" id="data">
+                      <option value="0">0</option>
+                      <option value="1">1</option>
+                      <option value="2">2</option>
+                      <option value="3">3</option>
+                    </select>
+                  <label for="object">Object manipulation</label>
+                    <select name="object" id="object">
+                      <option value="0">0</option>
+                      <option value="1">1</option>
+                      <option value="2">2</option>
+                      <option value="3">3</option>
+                    </select>
+                  <label for="dom">DOM manipulation</label>
+                    <select name="dom" id="dom">
+                      <option value="0">0</option>
+                      <option value="1">1</option>
+                      <option value="2">2</option>
+                      <option value="3">3</option>
+                    </select>
+                </li>
+                <li>
+                  <label for="funct">Functions</label>
+                    <select name="funct" id="funct">
+                      <option value="0">0</option>
+                      <option value="1">1</option>
+                      <option value="2">2</option>
+                      <option value="3">3</option>
+                    </select>
+                  <label for="temp">Templating</label>
+                    <select name="temp" id="temp">
+                      <option value="0">0</option>
+                      <option value="1">1</option>
+                      <option value="2">2</option>
+                      <option value="3">3</option>
+                    </select>
+                  <label for="eventh">Event Handling</label>
+                  <select name="eventh" id="eventh">
+                    <option value="0">0</option>
+                    <option value="1">1</option>
+                    <option value="2">2</option>
+                    <option value="3">3</option>
+                    </select>
+                </li>
+                <li>
+                  <label for="prot">Prototype & OOP</label>
```

Comment:
That's because GitHub uses Markdown syntax by default, which is, in a sense, a superset of HTML, and thus automatically encodes your entities to their corresponding character value. What you _can_ do, though, is use backtick (*\`*) characters around the string to escape the actual content, so that it's displayed like so: `&amp;`. Fun fact: you can use the `Preview` tab above to see what the end result will look like. :wink: